tests/test_pixel_breath.py

```diff
@@ -174,7 +174,7 @@ def _mock(*_args, **_kwargs) -> np.ndarray:
                 name="Tidal impedance variation",
                 unit=None,
                 category="impedance difference",
-                time=np.linspace(0, 100),
+                time=np.linspace(0, 100, 100),
```

Comment:
Another comment about the `_mock()` function. Do you need the `if`/`else`-block? Couldn't you use `values=np.full(100, np.sign(mean))`? `np.sign()` returns -1 if mean is negative, 0 if it is 0 and 1 if it is positive.